.github/workflows/run-tests.yml

```
@@ -0,0 +1,53 @@
+name: Run Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  run-core-js-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Main Repository
+        uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v3
+      - name: Install Main Dependencies
+        run: npm i
+      - name: Bootstrap Packages
+        run: npm run bootstrap
+      - name: Run Tests
+        working-directory: packages/eslint-config-core-js
+        run: npm test
+  run-core-ts-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Main Repository
+        uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v3
+      - name: Install Main Dependencies
+        run: npm i
+      - name: Bootstrap Packages
+        run: npm run bootstrap
+      - name: Run Tests
+        working-directory: packages/eslint-config-core-ts
+        run: npm test
+  run-react-js-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Main Repository
+        uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v3
+      - name: Install Main Dependencies
+        run: npm i
+      - name: Bootstrap Packages
+        run: npm run bootstrap
+      - name: Run Tests
+        working-directory: packages/eslint-config-react-js
+        run: npm test
```

As agreed, add [matrix](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs)

```suggestion
jobs:
  run-tests:
    strategy:
      matrix:
        package: [eslint-config-core-js, eslint-config-core-ts, eslint-config-react-js]

    runs-on: ubuntu-latest
    steps:
      - name: Checkout Main Repository
        uses: actions/checkout@v3
      - name: Setup Node
        uses: actions/setup-node@v3
      - name: Install Main Dependencies
        run: npm i
      - name: Bootstrap Packages
        run: npm run bootstrap
      - name: Run Tests
        working-directory: packages/${{ matrix.package }}
        run: npm test
```